storage-migration-error.asc

```diff
@@ -0,0 +1,54 @@
+:docinfo1:
+= Unexpected error during Storage Migration
+:sectanchors:
+
+There was an error copying your media to a folder supported by Android R. Before the process started, your data was backed up to AnkiWeb and can be restored.
+
+Before continuing, please:
+
+1. Take a backup of your collection (including scheduling + media) via Deck Picker -> Export collection
+  * See <<take_manual_backup,'Manual Backups'>> if you desire a file-based backup
```

Comment:
The collection in case of a failure is likely to be split between two folders, is exporting helpful?